books.md

```diff
@@ -4,6 +4,7 @@
 - I Shall Wear Midnight
 - Nightwatch
 - The Bromeliad
+- More books - I don't read!
```

Comment:
Are you sure? what about smoke and mirrors?

Reply:
Oh! I didn't think of that I will add that now 👍🏼